media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored the the FLAC format.  flac123 is the only \
```

Comment:
Lol that's actualy copied from readme inside the flac123 if i'm not mistaken XD.